Diagnostics/HealthChecker/Analyzer/Invoke-AnalyzerFrequentConfigurationIssues.ps1

```diff
@@ -349,6 +349,26 @@ function Invoke-AnalyzerFrequentConfigurationIssues {
         Add-AnalyzedResultInformation @params
     }
 
+    if ($null -ne $exchangeInformation.GetTransportService) {
+        if ($exchangeInformation.GetTransportService.MaxPerDomainOutboundConnections -lt 40) {
+            $params = $baseParams + @{
+                Name             = "MaxPerDomainOutboundConnections"
+                Details          = "Value set to $($exchangeInformation.GetTransportService.MaxPerDomainOutboundConnections), which is less than the recommended value of 40. `r`n`t`tMore details: https://aka.ms/HC-TransportRetryConfig"
```

Comment:
Did you create a different aka.ms link here? 

The one I provided was `https://aka.ms/HC-TransportRetryConfigCheck`